### DISCUSSION TITLE
I want to build the project from source but cannot find

### DISCUSSION THREAD

**Discussion by hououou** [memgraph/memgraph#364] [Q&A] [answered]
I want to build the project from source and I follow the step: https://memgraph.notion.site/Toolchain-37c37c84382149a58d09b2ccfcb410d7. But the link "https://s3-eu-west-1.amazonaws.com/deps.memgraph.io/toolchain-v4/toolchain-v4-binaries-centos-7-x86_64.tar.gz" cannot be found. Can you give me the correct link? Thank you

**Comment by jbajic** [comment, marked as answer]
Hmm, I have no issue downloading the toolchain with the given link, can you tell me the exact error you get, is it 404 or something else?

**Reply by hououou** [reply]
server "s3-eu-west-1.amazonaws.com". not found

**Reply by hououou** [reply]
I tried another network and made it. Thank you

**Reply by jbajic** [reply]
Aha great to hear that, if you have any other issues please let us know, and good luck with building :hammer:!

**Comment by gitbuda** [comment]
A quick note from my side 😄

Our community has enthusiastically embraced [Discord](https://discord.gg/memgraph) and [Stackoverflow](https://stackoverflow.com/questions/tagged/memgraphdb) as the favored channels for sharing with our team and asking questions. We’ve decided to disable the discussions tab here on GitHub in 2 weeks.

We look forward to seeing you there!
